nixos/modules/services/networking/ssh/sshd.nix

```
@@ -269,7 +269,7 @@ in
           ];
         example =
           [ { type = "rsa"; bits = 4096; path = "/etc/ssh/ssh_host_rsa_key"; rounds = 100; openSSHFormat = true; }
```

```suggestion
          [ { type = "rsa"; bits = 4096; path = "/etc/ssh/ssh_host_rsa_key"; openSSHFormat = true; }
```
Given this is also a host key, the same applies?